site/src/Librecores/ProjectRepoBundle/Entity/Project.php

```diff
@@ -312,6 +312,14 @@ class Project
      */
     private $releases;
 
+    /**
+     * Code quality metrics for the project
+     * @var float
+     *
+     * @ORM\Column(type="float", nullable=true)
+     */
+    private $score;
```

Comment:
could you change that attribute to `qualityScore` to make it a bit more descriptive?